src/app/body/all-products/product-card/product-card.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, Input, OnInit} from '@angular/core';
+// @ts-ignore
+import mockImg from 'src/assets/mockImg.svg';
+// @ts-ignore
+import starBlack from 'src/assets/star-black.svg';
+// @ts-ignore
+import star from 'src/assets/star.svg';
+// @ts-ignore
+import heart from 'src/assets/heart.svg';
+
+@Component({
+  selector: 'app-product-card',
+  templateUrl: './product-card.component.html',
+  styleUrls: ['./product-card.component.scss']
+})
+export class ProductCardComponent implements OnInit {
+  @Input() product;
```

Comment:
`@Input() product`
Where is interface for product?

Reply:
fixed